src/OpenDiffix.Core/CommonTypes.fs

```diff
@@ -206,18 +206,17 @@ type IAggregator =
 
 type AggregationContext =
   {
-    BaseExecutionContext: ExecutionContext
+    ExecutionContext: ExecutionContext
     GroupingLabels: Expression array
-    AggregatorFunctions: Function array
-    AggregatorArgs: array<Expression list>
+    Aggregators: (Function * Expression list) array
+    LowCountIndex: int option
```

Comment:
I meant that this should be put in the execution context so it doesn't have to be computed during aggregation.
Or is that more convoluted to do?

Reply:
The execution context is query (request) wide, whereas the aggregation context is node-specific. I think it's cleaner this way as it holds state more locally.

Reply:
Alternatively we can store it in the plan node. But this is a computed value from aggFns so it's redundant to carry around.

Reply:
> The execution context is query (request) wide

Good point!

Reply:
Or we don't pass it at all! The hooks can find it easily...

Reply:
Good idea! Even better to provide a utility function for that.